### PR TITLE
Neo4j timing CRUD operations (#4)

### PR DESCRIPTION
**Pull request by chinkitp**
CRUD Cypher queries

### CODE REVIEW

File: neo4j/src/main/java/Neo4jLoader.java [hunk 1]
```diff
@@ -1,237 +1,258 @@
...
+    public void UpdateRelationship(Map<String, Object> param) throws InvalidDataException {
 
-            return result1.single().get(0).asNode().toString();
-        });
+        String query = "MATCH p=()-[r:Lr41MAakQRqPwVuj3j4jCg]->() " +
+                       "WHERE r.id = $id " +
+                       "SET r.NewProperty = $value";
 
+        ResultSummary result = ExecuteScalar(query,param);
+        if(!result.counters().containsUpdates())
```
Comment (gaj67) [inline neo4j/src/main/java/Neo4jLoader.java]: Spacing around brackets, i.e. "if (" - I don't like the modern tendency to treat keywords like method calls!

File: neo4j/src/main/java/Neo4jLoader.java [hunk 2]
```diff
@@ -1,237 +1,258 @@
...
+            }
+        }
+    }
 
-        System.out.println("Creating head:");
-        System.out.println(query);
 
-        final String result = session.writeTransaction(tx -> {
-            StatementResult result1 = tx.run(query);
+    public void UpdateRelationship(Map<String, Object> param) throws InvalidDataException {
```
Comment (gaj67) [inline neo4j/src/main/java/Neo4jLoader.java]: Method names start with lower case - the standard is for only class names to start with initial capitalisation.

File: neo4j/src/main/java/Neo4jLoader.java [hunk 3]
```diff
@@ -1,237 +1,258 @@
...
 
-    private List<String> fileList(Path directory) throws IOException {
-        if (!Files.isDirectory(directory)) {
-            throw new IOException("Invalid directory");
-        }
+    public static void main( String... args ) throws Exception
+    {
+        try (Neo4jLoader neo4jLoader = new Neo4jLoader( "bolt://localhost:7687" ))
+        {
+            String NodeIdForSelect = "rkh0U1OGRi26ipTMu0NjtQ";
```
Comment (gaj67) [inline neo4j/src/main/java/Neo4jLoader.java]: Variable names start with lower case.

File: neo4j/src/main/java/Neo4jLoader.java [hunk 4]
```diff
@@ -1,237 +1,258 @@
...
+            String RelationshipIdForUpdateProperty = "loYE81RJRsu9KI7JFH1K7Q";
+
+            String RelationshipIdForDelete = "pxLD/IPATQyXqFfYpeBHpA";
+            String NodeIdForDelete = "L3hYub8eM2KV7FYXnSzR2w";
+
+            long start;
+
+            neo4jLoader.CreateIndex();
+
+            start = System.currentTimeMillis();
```
Comment (gaj67) [inline neo4j/src/main/java/Neo4jLoader.java]: Don't split declaration and initialisation.

**Comment by chinkitp** [comment]
Thanks @gaj67  code cleaned

**Pull request merged**